index.html

```diff
@@ -1,47 +1,770 @@
-<html>
-    <head>
-        <meta charset='UTF-8'/>
-        <title>Positionning and floating element</title>
-        <link rel="stylesheet" href="css/style.css">
-    </head>
-    <body>
-        <head>
-            <nav>
-                <ul class="navigation">
-                    <li><a href="#"></a></li>
-                    <li><a href="#"></a></li>
-                    <li><a href="#">SPACE & COSMOS</a></li>
-                    <li><a href="#"> <img src="" alt="logo The New York Time"> </a></li>
-                    <li><a href="#">Subscribe Now</a></li>
-                    <li><a href="#">Log In</a></li>
-                </ul>
-            </nav>
-        </head>
-        
-        <section>
-            <article>
-            </article>
-
-            <aside>
-            </aside>
-        </section>
-
-        <section>
-            <article>
-            </article>
-            <aside>
-            </aside>
-        </section>
-        
-
-
-        <footer>
-            <article>
-
-            </article>
-           <nav>
-           </nav>
-        </footer>
-        
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset='UTF-8' />
+    <title>Positionning and floating element</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://g1.nyt.com/fonts/css/web-fonts.5810def60210a2fa7d0848f37e3fa048bb6147b1.css" rel="stylesheet"
+        type="text/css">
+</head>
+
+<body>
+    <header>
+        <nav class="top">
+            <ul class="navigation">
+
+                <li class="menu"><i class="fa fa-bars"></i></li>
```

Comment:
It has a hamburger button
![hamb](https://user-images.githubusercontent.com/46388113/73402504-4b14a200-42a2-11ea-8f9b-4bd796af10e5.PNG)

Reply:
thanks for your observations